Few-iOS/TableView.swift

```diff
@@ -104,19 +106,40 @@ public class TableView: Element {
     public override func applyDiff(old: Element, realizedSelf: RealizedElement?) {
         super.applyDiff(old, realizedSelf: realizedSelf)
         
-        if let scrollView = realizedSelf?.view as? FewTableView {
-            let handler = scrollView.handler
+        if let tableView = realizedSelf?.view as? FewTableView {
+            let handler = tableView.handler
             
             layoutElements()
             
             handler?.elements = elements
             
-            let tableView = scrollView.handler?.tableView
-            
+            let newHeaderView = header?.realize(realizedSelf).view
```

Comment:
Ooof, yeah this gets weird because of the difference between Few's model and UIKit's :\

I *think* what we want to do is:

1. If it's the first time we're being rendered, realize the header element.
1. Keep the realized header view around.
1. Try to diff the realized header and the new header element (`canDiff` + `applyDiff`).
1. Perform layout of the element.
1. If the height changed, tell the table view.
1. Copy the previously realized header to the new `TableView` element.

In some ways, it's pretty similar to how Component works.